project-base/tests/ShopBundle/Acceptance/acceptance/PageObject/Front/CartBoxPage.php

```diff
@@ -4,15 +4,21 @@
 
 namespace Tests\ShopBundle\Acceptance\acceptance\PageObject\Front;
 
+use Shopsys\FrameworkBundle\Component\Money\Money;
 use Tests\ShopBundle\Acceptance\acceptance\PageObject\AbstractPage;
 
 class CartBoxPage extends AbstractPage
 {
     /**
-     * @param string $text
+     * @param int $expectedCount
+     * @param string $expectedPrice
      */
-    public function seeInCartBox($text)
+    public function seeInCartBox(int $expectedCount, string $expectedPrice): void
```

Comment:
this method name doesn't make sense anymore.. because you limit it only to the message and can't use it to search for everything in the cart ox as it was before

Reply:
renamed in 059e5a688